setup.py

```diff
@@ -51,7 +51,7 @@
     entry_points={
         'console_scripts': ['pdfarranger=pdfarranger.pdfarranger:main']
     },
-    install_requires=['pikepdf>=1.17.0','python-dateutil>=2.4.0'],
+    install_requires=['pikepdf>=2.14.0','python-dateutil>=2.4.0'],
```

Comment:
Are we ready to cut support for pikepdf < 2.14? According to repology that would include Debian 11. (It can probably be installed from pypi.)